lesson17/src/js/main.js

```diff
@@ -0,0 +1,225 @@
+function showLoadingImage() {
+  const main = document.getElementById("main");
+  const fragment = document.createDocumentFragment();
+  const renderLoadingBox = document.createElement("div");
+  renderLoadingBox.classList.add("loading");
+  renderLoadingBox.id = "js-loading";
+  const loadingImage = document.createElement("img");
+  loadingImage.src = "images/loading.gif";
+  loadingImage.width = 80;
+  loadingImage.height = 80;
+  loadingImage.classList.add("loading__image");
+  fragment.appendChild(renderLoadingBox).appendChild(loadingImage);
+  document.body.insertBefore(fragment, main);
+}
+
+function removeLoading() {
+  document.getElementById("js-loading").remove();
+}
+
+function displayErrorMassage(error) {
+  const slideImageArea = document.querySelector(".mainvisual__images__inner");
+  const createTextBox = document.createElement("p");
+  createTextBox.classList.add("");
+  createTextBox.textContent = error;
+  slideImageArea.appendChild(createTextBox);
+}
+
+function renderSlideArea() {
+  const slideImageArea = document.createElement("div");
+  slideImageArea.classList.add("mainvisual__images__inner");
+
+  const sliderArea = document.querySelector(".mainvisual__images");
+
+  sliderArea.appendChild(slideImageArea);
```

Comment:
IMO
Since the variable was used only once, I felt it was not necessary to declare it.
```suggestion
  document.querySelector(".mainvisual__images").appendChild(slideImageArea);
```

Reply:
Thank you for your review!
I put it together in one line!
This is easier to read.

[1cf8f9c](https://github.com/nyattsu72/hanson-frontend-cource/pull/25/commits/1cf8f9c032bca88e0de001c24945242b52847c41)